src/core/internal/lifetime.d

```diff
@@ -105,7 +108,8 @@ if (!is(T == const) && !is(T == immutable) && !is(T == inout))
             import core.stdc.string : memset;
             memset(cast(void*) &chunk, 0, T.sizeof);
         }
-        else static if (T.sizeof <= 16 && !hasElaborateAssign!T && __traits(compiles, (){ T chunk; chunk = T.init; }))
+        else static if (is(T == __vector) ||
```

Comment:
Doesn't work for enum types: https://run.dlang.io/is/aUNF83
```d
import core.simd : float4;
enum E : float { _ }
static assert(is(float4 == __vector));
static assert(is(E == __vector)); // fails
```

Reply:
You don't run out of ideas. :] - I'm done with this PR though, still enough other 2.097 issues to overcome for LDC.